index.html

```diff
@@ -831,10 +831,14 @@ <h3 id="miscellaneous"> Miscellaneous </h3>
     <td><img src='https://img.shields.io/gitter/room/nwjs/nw.js.svg?maxAge=2592000' alt=''/></td>
     <td><code>https://img.shields.io/gitter/room/nwjs/nw.js.svg</code></td>
   </tr>
-  <tr><th> JIRA issue: </th>
-    <td><img src='https://img.shields.io/jira/issue/https/issues.apache.org/jira/KAFKA-2896.svg?maxAge=2592000' alt=''/></td>
+  <tr><th data-keywords='jira-issues'> JIRA issue: </th>
+    <td><img src='/jira/issue/https/issues.apache.org/jira/KAFKA-2896.svg?maxAge=2592000' alt=''/></td>
```

Comment:
This was not generated. The `index.html` file is meant to be generated from `try.html`. Could you remove those changes please?